src/type-translator.ts

```diff
@@ -164,12 +164,24 @@ export class TypeTranslator {
    * - TypeChecker.typeToString translates Array as T[].
    * - TypeChecker.symbolToString emits types without their namespace,
    *   and doesn't let you pass the flag to control that.
+   * @param useFqn whether to scope the name using its fully qualified name.
    */
-  public symbolToString(sym: ts.Symbol): string {
+  public symbolToString(sym: ts.Symbol, useFqn: boolean): string {
```

Comment:
I've recently preferred to avoid optional arguments. I often have the impression that it's better to be explicit at callsites as to what functionality you want, rather than have things work some way by default.

E.g. here, I think it makes sense that you need to think about whether a name should be qualified or not if you're adding new code calling into it.